Makefile

```diff
@@ -40,46 +46,21 @@ clean: # Clean up project
 
 trust-certificate: ssl-trust-certificate-project ## Trust the SSL development certificate
 
-# ==============================================================================
-# Pipeline targets
-
-build-artefact:
-	echo TODO: $(@)
-
-publish-artefact:
-	echo TODO: $(@)
-
-backup-data:
-	echo TODO: $(@)
-
-provision-infractructure:
-	echo TODO: $(@)
-
-deploy-artefact:
-	echo TODO: $(@)
-
-apply-data-changes:
-	echo TODO: $(@)
-
 # --------------------------------------
 
-run-static-analisys:
-	echo TODO: $(@)
-
 run-unit-test: # Run unit tests, add NAME="xxx" or NAME="xxx or yyy" to run specific tests
-	if [ -z $(TEST_FILE) ]; then
-		docker exec roaddistance-lambda \
-			/bin/sh -c 'for f in tests/unit/test_*.py; do python -m pytest -rsx -q $$f; done'
-	else
-		docker exec roaddistance-lambda \
-			python -m pytest -rA -q tests/unit/$(TEST_FILE) -k "$(NAME)"
-	fi
-
-run-smoke-test:
-	echo TODO: $(@)
-
-run-integration-test:
-	echo TODO: $(@)
+		if [ $(BUILD_ID) == 0 ]; then
```

Comment:
This seems to depend on a running container. Is there a need for that? Could we instead use `make docker-run IMAGE=...`?

Reply:
Thanks, will take a look